src/main/java/com/example/may/email/service/EmailService.java

```diff
@@ -19,38 +20,31 @@
  * @author Evelina Tubalets
  */
 @Service
+@AllArgsConstructor
 public class EmailService {
 
-    private final String host;
     private final UserRepository userRepository;
     private final EmailProperties properties;
-
-    public EmailService(
-            @Value("${spring.mail.host}") final String host,
-            final UserRepository userRepository,
-            final EmailProperties properties) {
-        this.host = host;
-        this.userRepository = userRepository;
-        this.properties = properties;
-    }
+    private final DestinationService propertyReceiver;
 
     public void sendCongratulationEmail() {
         final List<User> birthdayUsers = getBirthdayUserEmails();
         birthdayUsers.forEach(this::createCongratulationMessageAndSend);
     }
 
+    public void sendTestEmail() {
+        final SimpleMailMessage testMessage = createTestMessage();
+        getMailSender().send(testMessage);
```

Comment:
firstly get instance of sender, then call it (try to avoid calls like a().b().c().d()...):
```
final SimpleMailMessage mailSender = getMailSender();
mailSender.send(testMessage);
```